Lib/Salesforce/SalesforceAuthManager.php

```diff
@@ -92,11 +95,14 @@ public function getAuthContainer($userId) {
 		$salesforce = $this->salesforce;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "$salesforce" which has the same name as the field declared at line 17. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS1117)